todo.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script type="text/javascript">
+</script>
+</head>
+
+<body>
+<h1>Todo List</h1>
+    <div id="todoList"></div>
```

Comment:
Semantics in HTML are really important since it's not a programming language but more of a declarative document format. In this case you have a list, so the appropriate tag type would be a `UL` and the nested elements would be `LI`s